GNUmakefile

```diff
@@ -47,7 +47,7 @@ build: init
 ## RPM-related targets (deprecated; use targets in rpm subdirectory instead)
 
 # Pass option depending on whether automake has been run or not
-USE_FILE	= $(shell test -z "$(srcdir)" && echo "-f Makefile.am")
+USE_FILE = $(shell test -e rpm/Makefile || echo "-f Makefile.am")
 
```

Comment:
hmm ... up to now I could update the git-tree and wouldn't have to care if there were leftover local files or not when e.g. building a src-rpm. Now an existing rpm/Makefile makes it use that even if outdated.
I'm meanwhile using rpm/Makefile.am for that purpose directly but previously I had been using GNUmakefile as long as the targets were directly in there.

Reply:
Actually I don't think this makes it any worse. If there were artifacts from a previous build, the original -z test wouldn't hit because the top-level Makefile would also exist. The new test could be a problem only if rpm/Makefile existed but the top-level Makefile didn't.

Reply:
Of course that's just compared to the immediate previous code -- the problem you described probably started when the rpm stuff was pulled out of GNUmakefile. I don't see a better approach, though.

Reply:
Not seeing anything nice and easy either - otherwise I'd have mentioned it.
To a certain extent I guess it makes it worse. For the case that sbdy is using GNUmakefile directly - which I did up to some changes that made it not work anymore for me (probably would work now again) - it would recognize that it isn't called from out of the Makefile-hierarchy (no $(srcdir) defined) and thus go for Makefile.am directly which is under source-control and thus updates on a git pull.
But as long as you can go for rpm/Makefile.am directly to be on the safe side I guess the change is fine.
Maybe just something to remember if issues should be coming up ...

Reply:
And yes I guess issues date back to when moving the stuff to the rpm-subdir. But I don't remember the details.
Guess at some point using the targets from GNUmakefile directly was broken - which is when I obviously moved to using rpm/Makefile.am - and I have some vague memory that the current thing tried to make that work again.